packages/flutter/test/material/ink_paint_test.dart

```diff
@@ -434,4 +434,35 @@ void main() {
       throw 'Expected: paint.color.alpha == 0, found: ${paint.color.alpha}';
     }));
   });
-}
+
+  testWidgets('Does the Ink widget render anything if it have ancestor IndexedStack', (WidgetTester tester) async {
+    int index = 0;
```

Comment:
Can you add a `Regressing test for https://github.com/flutter/flutter/issues/59963`